lesson04/assignment/css/style.css

```diff
@@ -0,0 +1,362 @@
+        'sidebar sidebar sidebar sidebar sidebar'
+        'article article article article article'
+        'footer footer footer footer footer';
+}
+.item1 {
+    padding: 0;
+    color: black;
+    font-size: 1.5em;
+    text-align: left;
+}
+.item2 {
+    grid-area: header;
+    text-align: left;
+    padding-left: 1.5em;
+    padding-top: .5em;
+}
+    /* Header */
+    /* Header image */
+    h2 h3 {
+
+    }
+    img {
+      display: block;
+      width: 100%;
+      height:50%;
+      object-fit:cover;
+    }
+    /* Header background section */
+    header svg {
+      width:100%;
+      height: 100vh;
+    }
+    .img-overlay-wrap img {
+      position: absolute;
+      max-width: 100%;
+      opacity: 0.5;
+      top: 0;
+      left: 0;
+      align: center;
+      z-index: -1;
+    }
+    .img-overlay-wrap svg {
+      position: absolute;
+      verticle-align: center;
+    }
+    /* End Header background section */
+
+/* Master navigation */
+.itemN {
+    grid-area: nav;
+    list-style-type: none;
+}
+    /* Web Design Playground: wdpg.io/projects/portfolio-page/04 */
+    h1.itemN {
+      padding-top: .1em;
+    }
+    .btn {
+      background-color: dimgray;
+      padding: .25rem .75rem;
+      border-radius: .75rem;
+      font-size: .5em;
+      color: lightgray;
+      text-transform: uppercase;
+    }
+    nav {
+      z-index: 10;
+      position: sticky;
+      height: 3.5em;
+      width: 100%;
+    }
+    nav ul {
+      display: flex;
+      list-style-type: none;
+    }
+    nav li {
+      padding-right: 1rem;
+      padding-top: .5rem;
+
+    }
+    a {
+      text-decoration: none;
+      transition: all 0.5s ease-out;
+    }
+    a:hover,
+    a:focus {
+      color: dimgray;
+      background-color: lightgray;
+    }
+    /* End master navigation */
+/* End Header */
+
+/* Start of Main Content */
+.item3 {
+    grid-area: main;
+}
+/* Introduction with box transitions and transform */
+.intro {
+  display: flex;
+  justify-content:space-around;
+  align-items: center;
+  margin: 0;
+  padding: 0;
+  width: 100%;
+  color: dimgray;
+  font-size: 1.25rem;
+  background-image: radial-gradient(
+  ellipse closest-corner at center, #fffff0 25%, #ff8c00 100%);
+}
+.intro-text {
+  display: block;
+  padding: 2rem 0 2rem 2rem;
+  text-align: center;
+}
+.intro-text p {
+  margin: 1.5em 0;
+}
+.intro-image {
+  display: flex;
+  flex-shrink: 2;
+  padding: 0 1rem 0 1rem;
+  text-align:center;
+}
+h1 {
+  font-size: 3em;
+  font-weight: bold;
+  font-family:serif;
+}
+
+/* End of Introduction with box transitions and transform */
+/* division transition and transform */
+/* https://www.w3schools.com/css/tryit.asp?filename=trycss3_transition_transform */
+ #box {
+  border-color: midnightblue;
+  border-style: solid;
+  border-width: 2em, 2em, 2em, 2em;
+  margin: .5em;
+  display: flex;
+  width:304px;
+  height: 154px;
+  background-color: darkorange;
+  transition: width 3s, height 3s, transform 1s
```

Comment:
Using css translate and scale will result in smoother looking animations and better performance than using height/width.  Have a look at this link for an example:
https://pqina.nl/blog/animating-width-and-height-without-the-squish-effect/